lua/tardis/parts/legacy/legacy_helmic.lua

```diff
@@ -0,0 +1,19 @@
+--Legacy Interior - Helmic Regulator
+
+local PART = {}
+PART.ID = "legacy_helmic"
+PART.Name = "Legacy Helmic Regulator"
+PART.Model = "models/drmatt/tardis/helmicregulator.mdl"
+PART.AutoSetup = true
+PART.Collision = true
+PART.Animate = true
+PART.AutomaticFrameAdvance = true
+
+if SERVER then
```

Comment:
Remove section

Reply:
Actually looks like this used to control spin direction in legacy

https://github.com/MattJeanes/TARDIS/blob/legacy/lua/entities/sent_tardis_helmicregulator/init.lua

Reply:
Wait no it didn't that was just the animation nvm